### PR TITLE
README - move running local demo to Contributing

### PR DESCRIPTION
**Pull request by piotr-s-brainhub**
Deployed to https://beghp.github.io/gh-pages-rc-5<br><br>- [ ] an issue linked to the PR<br><br>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,6 +197,29 @@ You're also more than welcome to:
 - comment an issue, discussing the details
 - open a PR, fixing a given issue
 
+### Setting up local development which means running the docs/demo locally:
```
Comment (humbak) [inline README.md:200]: Why you moved local set up from installation to contributing? I don't understand this decision.

File: README.md [hunk 2]
```diff
@@ -214,6 +197,29 @@ You're also more than welcome to:
...
+- `git clone https://github.com/brainhubeu/react-carousel`
+- `cd react-carousel`
+- `yarn`
+- `cd docs-www`
+- `yarn`
+- if you want to connect demo with the carousel source code, replace `__RC_ENV__` into `development` in https://github.com/brainhubeu/react-carousel/blob/master/docs-www/src/globalReferences.js#L2 and remove the `.babelrc` file in the root directory; otherwise, it will use the carousel code installed in `docs-www/node_modules`
+- `yarn develop`
+- open http://localhost:8000/
+
+### Tests
```
Comment (humbak) [inline README.md:210]: Please explain to me this decision. I don't understand it.

File: README.md (hunk 1 above)
Comment (piotr-s-brainhub) [reply README.md:200]: only contributors need to set up docs locally and run tests;

people, who want only use carousel, can click the deployed docs and see the tests result in CI

Comment (humbak) [reply README.md:200]: ok you convinced me

**Pull request merged**